src/cdk/explorer/tree/types.ts

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Represents the CDK construct tree
+ */
+export interface ConstructTree {
```

Comment:
This (and tree entity) appear to be some of your most core structures. Please comment the members to share context with other people reading the code.